app/assets/stylesheets/refinery/stories.scss

```diff
@@ -1,5 +1,44 @@
 .StoriesPage {
 
+  .weigh-in__header {
+    display: flex;
+    background-color: $color_bg-lightest;
+    background-image: url('group-2111.png');
+    padding: 14.18rem 19.5rem 14.18rem 37.43rem;
+    
+    @media (max-width: 1600px){
```

Comment:
Instead of using defined pixel values here maybe when you refactor to use container in the DOM, you can implement using the breakpoints we use in `_vars.scss` https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/global/_vars.scss#L63-L67. See https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/refinery/home.scss#L61-L109 for an example of how this is implemented. If standardizing these break points turns into a bear I'm happy to pair on this with you tomorrow morning.